cmd/repository/default-pkg-registry-entry.go

```diff
@@ -0,0 +1,25 @@
+package repository
+
+import "github.com/criteo/command-launcher/internal/command"
+
+type defaultRegistryEntry struct {
```

Comment:
Is this registry entry still in use? I couldn't find any reference anymore in the review. If it is the case, we should remove it